rclpy/rclpy/impl/logging_severity.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
```

Comment:
```suggestion
# Copyright 2021 Open Source Robotics Foundation, Inc.
```

Reply:
Well, given that this lifts existing code from one place and puts it in another, I figure that copying the old date makes sense.

Honestly, it probably doesn't matter much one way or another; we don't have a policy of updating the date when we fix code anyway.